### DISCUSSION TITLE
Widget should open clock app without changing tab

### DISCUSSION THREAD

**Discussion by ritlew** [breezy-weather/breezy-weather#1043] [Ideas]
### Describe your idea

I use the widget to frequently open the clock app. Usually I'm trying to set or look at timers. The widget will always open the "Alarms" page instead of just opening the app in its current state. The widget should open the app to however it was left.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.3 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
We can't do that, it's a standard Android behavior, we just call the Alarm intent, as other clock widgets do

**Reply by ritlew** [reply]
https://developer.android.com/guide/components/intents-common#CreateTimer

**Reply by papjul** [reply]
This is to create a timer (countdown) in your clock app, not to reopen last opened tab of the clock app

**Reply by ritlew** [reply]
Yep. It would be nice to switch to this in the widget options
